egs/vcc2020/voc1/conf/parallel_wavegan.task2.yaml

```diff
@@ -0,0 +1,120 @@
+# This is the hyperparameter configuration file for Parallel WaveGAN.
+# This configuration requires 12 GB GPU memory and takes ~3 days on TITAN V.
+
+###########################################################
+#                FEATURE EXTRACTION SETTING               #
+###########################################################
+sampling_rate: 24000     # Sampling rate.
+fft_size: 1024           # FFT size.
+hop_size: 384            # Hop size.
```

Comment:
Is this parameter intended?

Reply:
In all TTS model feature extraction, the `hop_size` is 256 and the waveform is 16kHz,
so I set `hop_size` to 384 here so that we can upsample the 16kHz fbanks to 24kHz waveform.

But I think there should be some problem here.
In PWG feature extraction, we first downsample 24kHz to 16kHz, so here hop size should be still 256.

I wrote some custom codes in my own PWG experiment directory to overcome these problems,
but, anyway, can you implement some option so that we can make <24kHz waveform, 16kHz mel fbank>?
Maybe `parallel-wavegan-preprocess` can take two kinds of wav scps, one is for ground truth (24kHz), and another one is the downsampled one for feature extraction (16kHz). In the config file, we can also split the `hop_size` into two, one for feature extraction and one for checking the upsample scale of the generator.
Sorry for asking you to implement such complicated feature...

Reply:
OK. Let me confirm some points
- The target waveform is 24kHz
- In feature extraction, downsample from 24kHz -> 16kHz
- Instead of the hopsize in 24 kHz, hopsize * 16 kHz / 24 kHz will be used
Is it correct? If so, I will support sampling_rate_in_mel_ext option in config. (the hopsize will be calculated automatically and wav is downsampled in the scripts.)

Reply:
@unilight I implemented such option.
https://github.com/kan-bayashi/ParallelWaveGAN/pull/99
Could you check it? If OK, I will update version and upload to pypi.

Reply:
@kan-bayashi Thank you very much! It works fine. Please upload to pypi.

Reply:
Uploaded. 0.3.4 is available.